test/release/examples/primers/taskParallel.chpl

```diff
@@ -97,3 +97,72 @@ coforall i in 1..n {
 // The order of output is undefined.
 writeln("5: output from main task");
 
+/*
+.. _primers-taskparallel-task-intents:
+
+Task Intents
+------------
+
+The body of a task construct name some variables declared outside,
+or "outer variables". If so, "shadow variables" are introduced.
+Each task created by the task construct gets its own set of
+shadow variables, one per outer variable.
+
+ - Each shadow variable behaves as if it were a formal argument
+   of the task function for the task. The outer variable is passed
+   to this formal argument according to the argument intent
+   associated with the shadow variable, which is called a "task intent".
+
+ - The name of an outer variable lexically in the task construct
+   implicitly refers to the corresonding shadow variable.
```

Comment:
"References to the outer variable from within the task's body refer instead to the corresponding shadow variable." (?)

Reply:
> References to the outer variable from within the task's body refer instead to the corresponding shadow variable

Indeed this is what I was battling. "So, after all, does it refer to the outer variable or the shadow variable?" Perhaps we do not have to have absolute precision here, so I can make this change.

Reply:
One other attempt for consideration:  "References within a task that seem to refer to a variable outside of the task's lexical scope will actually be referring to the corresponding shadow variable owned by the task."

Reply:
I applied Brad's second suggestion above.